### PR TITLE
Mar 114/add a barrier step 1

### PR DESCRIPTION
**Pull request by fericsepi**

### CODE REVIEW

File: reports/urls.py [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+    NewReportBarrierLocation,
+)
 
 app_name = "reports"
 
 urlpatterns = [
     path("reports/new/", NewReport.as_view(), name="new_report"),
-    path("reports/new/start/", NewReportBarrierStatus.as_view(), name="barrier_status"),
+    path("reports/new/start/", NewReportBarrierStatus1.as_view(), name="barrier_status_1"),
+    path("reports/new/start/is-resolved/", NewReportBarrierStatus2.as_view(), name="barrier_status_2"),
```
Comment (dankingston) [inline reports/urls.py]: I found it hard to work out what the difference was between status1 and status2... maybe we could use more descriptive names? Confusingly the frontend seems to call both of these 'statuses', but in the API they are called 'problem_status' and 'status', so could go with that? I used those names for the edit views (`BarrierEditProblemStatus` and `BarrierEditStatus`).

File: core/apps.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class InteractionsConfig(AppConfig):
```
Comment (dankingston) [inline core/apps.py]: Guessing this should be `CoreConfig`

Comment (fericsepi) [reply core/apps.py]: oh yeah :) nice catch

File: reports/urls.py (hunk 1 above)
Comment (fericsepi) [reply reports/urls.py]: Same here, so until we work out what to call barrier type I sort of named them `1` and `2` after the sub-steps of Step 1 of adding a barrier which is titled `Barrier Status`.

Happy to change it so it matches the API namings.

**Comment by fericsepi** [comment]
note: rebased to resolve conflict

**Pull request merged**